distribution/openhabhome/configurations/openhab_default.cfg

```diff
@@ -1323,6 +1323,12 @@ tcp:refreshinterval=250
 # The Netatmo refresh token (see http://dev.netatmo.com/doc/authentication/usercred)
 #netatmo:refreshtoken=
 
+# The Netatmo unit system where M=Metric system, US=US System (optional, defaults to M)
```

Comment:
Would be good to explicitly document that US gives Fahrenheit, inches; M gives Celsius, mm, or whatever it gives, so the user doesn't have to guess.  Better would be to give the user control over each kind of unit, since they may want inches and Celsius, which is not an option.